lib/class-wp-rest-global-styles-controller-gutenberg.php

```diff
@@ -530,7 +530,7 @@ public function get_item_schema() {
 		}
 
 		$schema = array(
-			'$schema'    => 'http://json-schema.org/draft-04/schema#',
```

Comment:
I don't think we can update the controller schemas. The [code for validating REST API schemas](https://github.com/WordPress/wordpress-develop/blob/e8fed338f8f27363cb24f0ea2792cfa3b311b612/src/wp-includes/rest-api.php#L2071) only supports draft-04.

Let's just stick with the code in `schemas`.